src/bundle/Core/Command/Indexer/ContentIdListGeneratorStrategyInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Core\Command\Indexer;
+
+use Generator;
+use Symfony\Component\Console\Input\InputInterface;
+
+/**
+ * @internal
+ */
+interface ContentIdListGeneratorStrategyInterface
+{
+    /**
+     * @return \Generator<int, int[]>
```

Comment:
Make Andrew happy. Phpstorm for some reason doesn't recognize the original notation properly.

```suggestion
     * @return \Generator<int, array<int>>
```

Reply:
Interesting, I forgot about it. This must be in some follow-up branches. Thx :D